docs/design/access.md

```diff
@@ -83,6 +83,35 @@ K8s Cluster assets:
 
 This document is primarily about protecting K8s User assets and K8s cluster assets from other K8s Users and K8s Project and Cluster Admins.
 
+<!-- BEGIN MUNGE: EXAMPLE examples/guestbook/frontend-controller.yaml -->
+
+```yaml
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: frontend
+  labels:
+    name: frontend
+spec:
+  replicas: 3
+  selector:
+    name: frontend
+  template:
+    metadata:
+      labels:
+        name: frontend
+    spec:
+      containers:
+      - name: php-redis
+        image: kubernetes/example-guestbook-php-redis:v2
+        ports:
+        - containerPort: 80
+
+```
+
+[Download example](http://releases.k8s.io/HEAD/examples/guestbook/frontend-controller.yaml)
```

Comment:
I agree.

HOWEVER, it makes me think that all references outside of of /docs and /examples should be prefixed with `http://releases.k8s.io/HEAD/`, rather than using relative links. That should make the links work from gh_pages.